x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_lifecycle/ilm_summary.tsx

```diff
@@ -56,7 +59,8 @@ export function IlmSummary({
         signal,
       });
     },
-    [streamsRepositoryClient, definition]
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [streamsRepositoryClient, definition, stats]
```

Comment:
passing the stats reference here is a hack to trigger the a refresh of the summary whenever the ingestion rate graph is refreshed